dotcom-rendering/src/web/components/DynamicSlowMPU.tsx

```diff
@@ -0,0 +1,382 @@
+/* eslint-disable @typescript-eslint/naming-convention -- because underscores work here*/
+import { Hide } from '@guardian/source-react-components';
+import type { DCRContainerPalette, DCRGroupedTrails } from '../../types/front';
+import { AdSlot } from './AdSlot';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+import { FrontCard } from './FrontCard';
+
+type Props = {
+	groupedTrails: DCRGroupedTrails;
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+/* .___________.___________.___________.
+ * |###########|___________|           |
+ * |           |___________|    MPU    |
+ * |___________|___________|___________|
+ */
+const Card33_ColumnOfThree33_Ad33 = ({
+	cards,
+	containerPalette,
+	showAge,
+}: {
+	cards: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+}) => (
+	<UL direction="row">
+		<LI percentage="33.333%">
+			<FrontCard
+				trail={cards[0]}
+				containerPalette={containerPalette}
+				showAge={showAge}
+			/>
+		</LI>
+		<LI percentage="33.333%">
+			<UL direction="column">
+				<LI padSides={true} padBottom={true}>
+					<FrontCard
+						trail={cards[0]}
+						containerPalette={containerPalette}
+						showAge={showAge}
+						imageUrl={undefined}
+						headlineSize="small"
+					/>
+				</LI>
+				<LI padSides={true} padBottom={true}>
+					<FrontCard
+						trail={cards[1]}
+						containerPalette={containerPalette}
+						showAge={showAge}
+						imageUrl={undefined}
+						headlineSize="small"
+					/>
+				</LI>
+				<LI padSides={true}>
+					<FrontCard
+						trail={cards[2]}
+						containerPalette={containerPalette}
+						showAge={showAge}
+						imageUrl={undefined}
+						headlineSize="small"
+					/>
+				</LI>
+			</UL>
+		</LI>
+		<LI percentage="33.333%">
+			<Hide until="tablet">
+				{/* TODO: Replace mostpop with a more appropriate value */}
+				<AdSlot position="mostpop" />
+			</Hide>
+		</LI>
+	</UL>
+);
+
+/* ._________________._________________.
```

Comment:
just an aesthetic preference, but what do you think of this style?

<img width="256" alt="image" src="https://user-images.githubusercontent.com/37048459/183454856-ee234313-01f4-42af-8e72-182e1c1bef01.png">

Happy to make them if you like them, but I don't want to impose if not.

Reply:
either way, +1 on having the ascii diagrams! I find them really helpful

Reply:
> just an aesthetic preference, but what do you think of this style?
> 
> <img alt="image" width="256" src="https://user-images.githubusercontent.com/37048459/183454856-ee234313-01f4-42af-8e72-182e1c1bef01.png">
> 
> Happy to make them if you like them, but I don't want to impose if not.

Super cool! How did you do that?

Reply:
Using this set of characters:

```
    ┌─┬┐ top
    │ ││ head
    ├─┼┤ head_row
    │ ││ mid
    ├─┼┤ row
    ├─┼┤ foot_row
    │ ││ foot
    └─┴┘ bottom
```
I think I originally came across them when I was looking into the [rich](https://github.com/Textualize/rich/blob/a6f2d84511cda7f367dca527177d7b586264755b/rich/box.py) package for Python.

They don't render great in all fonts, but they seem to be fine in most monospace, including in [comments rendered in github](https://github.com/guardian/dotcom-rendering/blob/1be769c18136cab1785e93fec513a00e95ccd8ad/dotcom-rendering/src/web/components/ExtraCardsContainer.tsx#L25-L33).

Reply:
thought I'd open a quick [PR](https://github.com/guardian/dotcom-rendering/pull/5627) while I was thinking about this, though it'd probably need a bit of tweaking. Totally up to you whether you want to use them though :)

Reply:
These versions of the diagrams defo look clicker but I struggled to represent the image position. Do you have any tricks for indicating where the image sits? I tried this but it looks a bit off

```
/*
 * ┌───────────┐┌───────────┐
 * ├──###──────┤│    MPU    │
 * ├──###──────┤│           │
 * └──###──────┘└───────────┘
 */
```

Reply:
ah, good point... maybe something to leave for a potential refactor then? but I'll have a think and let you know if I come up with a promising solution